cclient/serialization/json/tests/get_node_info.c

```diff
@@ -68,12 +69,16 @@ void test_deserialize_get_node_info(void) {
   flex_trits_from_trytes(hash, NUM_TRITS_HASH, (const tryte_t*)TEST_81_TRYTES_3, NUM_TRYTES_HASH, NUM_TRYTES_HASH);
   TEST_ASSERT_EQUAL_MEMORY(hash, node_info->coordinator_address, FLEX_TRIT_SIZE_243);
 
+  serializer.vtable.get_node_info_serialize_response(&serializer, node_info, serializer_out);
+  TEST_ASSERT_EQUAL_STRING(json_text, serializer_out->data);
+
   get_node_info_res_free(&node_info);
+  char_buffer_free(serializer_out);
 }
 int main(void) {
   UNITY_BEGIN();
 
-  RUN_TEST(test_serialize_get_node_info);
-  RUN_TEST(test_deserialize_get_node_info);
+  RUN_TEST(test_request);
```

Comment:
Could we use `test_serialize_get_node_info` and `test_deserialize_get_node_info` for the consistncy?